mainWidget/main_widget_FloorsAndRooms.yaml

```diff
@@ -87,8 +87,8 @@ slots:
                                       groupItem: =loop.equipmentItem.name
                                       RollerItem: =loop.shutterItem.name
                                       RollerMode: = loop.modeItem.name
-                                      Title: "=loop.equipmentItem.metadata.uiSemantics ? loop.equipmentItem.metadata.uiSemantics.config.equipment + loop.equipmentItem.metadata.uiSemantics.config.preposition + loop.equipmentItem.metadata.uiSemantics.config.location : loop.equipmentItem.label"
-                                      visible: '=!vars.objVar ? false : !vars.objVar.selectThing ? false : (vars.objVar.selectThing=="Rollers") ? true : false'
+                                      Title: "=loop.shutterItem.metadata.uiSemantics ? loop.shutterItem.metadata.uiSemantics.config.equipment + loop.shutterItem.metadata.uiSemantics.config.preposition + loop.shutterItem.metadata.uiSemantics.config.location : loop.shutterItem.label"
+                                      visible: '=!vars.objVar ? false : !vars.objVar.selectThing ? true : (vars.objVar.selectThing=="Rollers") ? true : false'
```

Comment:
What would be the benefit here ? Don't see the need for changing from equipment item to shutter item.
And please do not revert changes I have made on purpose without asking.
The reason for changing visibility was performance issues. This was also included in the comment for that change.

Reply:
So finally I got it right - I shouldn't work late at night :-(